data/releases.yaml

```diff
@@ -574,6 +574,21 @@ Releases:
       repo: nats_ex
       link: https://github.com/iamd3vil/nats_ex
 
+- name: Common Lisp
+  type: nats-client
+  supported: false
+  language: Common Lisp
+  author:
+    - name: ccQ
+      link: https://github.com/ccqpein
+  version_num: 0.0.0
```

Comment:
Probably should have at least one number, even if 0.0.1. But would probably say first one should be 0.1.0.

Reply:
Just add version `0.1.0`, please check it out